ci-operator/config/openshift-knative/kn-plugin-func/openshift-knative-kn-plugin-func-release-next.yaml

```diff
@@ -22,6 +18,26 @@ tests:
   commands: GOFLAGS="" make test
   container:
     from: src
+- as: e2e-oncluster-test
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: serverless-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.15"
+  steps:
+    test:
+    - as: test
+      commands: |
+        openshift/cluster-prepare.sh
+        openshift/e2e_oncluster_tests.sh
```

Comment:
If you have multiple scripts like that, maybe they can be executed in different steps (if the first one doesn't modify environment variables in some way). That may increase visibility in case of failure.

Reply:
Agree, however o nthe cluster-prepare it creates a pod required on the next step. by switching it to another step a different namespace is create and the pod reference is lost. So I'm inclined to keep that way for now.